pkg/projects/Microsoft.NETCore.App/netcoreapp1.0.props

```diff
@@ -5,10 +5,6 @@
       <Version>1.3.0</Version>
       <Exclude>Compile</Exclude>
     </NETCoreApp10Dependency>
-    <NETCoreApp10Dependency Include="Microsoft.CodeAnalysis.VisualBasic">
```

Comment:
Nothing. As we chatted I'm going to keep those dependencies for the older target frameworks.